### ISSUE TITLE
[feautre request] Changing the color of the circle

### ISSUE THREAD

**Issue by David583** [lxqt/lxqt-powermanagement#234]
So, I love this circle style, but green color doesn't really fit in my desktop style. How about adding a feature that let's the user pick colors? Btw., I took a fast look at the code, I can change and re-complie, yeah?

**Comment by tsujan** [comment]
This is the bug tracker.

**Comment by David583** [comment]
No, I am talking about the color of the original icon. When you are calling the IconProducer::buildCircleIcon, the style='stop-color:rgb(125,255,125)  can be changed by the user.

**Comment by tsujan** [comment]
Sorry for my misunderstanding! This is a valid feature request. Reopening...
